Mollie.Api/Client/OrderClient.cs

```diff
@@ -55,5 +60,27 @@ public class OrderClient : BaseMollieClient, IOrderClient {
         public async Task<ListResponse<RefundResponse>> GetOrderRefundListAsync(string orderId, string from = null, int? limit = null) {
             return await this.GetListAsync<ListResponse<RefundResponse>>($"orders/{orderId}/refunds", from, limit).ConfigureAwait(false);
         }
+        private Dictionary<string, string> BuildEmbedParameter(bool embedPayments = false, bool embedRefunds = false)
+        {
+            var result = new Dictionary<string, string>();
+
+            var embedList = new List<string>();
+            if (embedPayments)
+            {
+                embedList.Add("payments");
+            }
+
+            if (embedRefunds)
+            {
+                embedList.Add("refunds");
+            }
+
+            if (embedList.Any())
+            {
+                result.Add("embed", string.Join(",", embedList));
```

Comment:
Can you use the `ToQueryString` method of the `DictionaryExtensions` class for this instead? This also makes sure any parameters are properly URL encoded and I'd like to use this method to build all the query parameters everywhere.

Reply:
`ToQueryString` is done within `GetOrderAsync` in `OrderClient.cs`. So the parameters will be encoded properly. 
````c#
return await this.GetAsync<OrderResponse>($"orders/{orderId}{embeds.ToQueryString()}").ConfigureAwait(false);
````
https://github.com/Viincenttt/MollieApi/pull/144/files/efb2f9af8578859c7aed77bbec21aa5376df7f82#diff-3588f8bc98aa7a12afccceb1de378287R25

I guess we can move the `ToQueryString` to the `BuildEmbedParameter` method, but since the embed parameter should be comma seperated there is still need for some conditionals. Or do you have a different idea?

Reply:
@Viincenttt Could you check on https://github.com/Viincenttt/MollieApi/pull/144#discussion_r325054875?

Reply:
@Viincenttt Is this already implemented somewhere else? I will be more than happy to make the requested changes, but  I do not know exactly what you means, since the `ToQueryString` is already being used.